dspace/config/dspace.cfg

```diff
@@ -1335,8 +1337,8 @@ websvc.opensearch.samplequery = photosynthesis
 # tags used to describe search service
 websvc.opensearch.tags = IR DSpace
 # result formats offered - use 1 or more comma-separated from: html,atom,rss
-# NB: html is not supported in DSpace7, use normal search module instead
-websvc.opensearch.formats = atom,rss
+# html uses the normal search module
+websvc.opensearch.formats = html,atom,rss
```

Comment:
I'm not able to figure out how to get HTML format to work properly.  It looks like we have a disabled HTML test here: https://github.com/DSpace/DSpace/blob/main/dspace-server-webapp/src/test/java/org/dspace/app/opensearch/OpenSearchControllerIT.java#L60 

However, if I enable that test, it fails.

Reply:
thanks to spot that. Indeed, the test is wrong. The html format is managed by the UI, so the test should be eventually replaced by an e2e test. I have removed the test and added a comment about it